src/common/error.rs

```diff
@@ -82,6 +82,28 @@ pub enum Error {
     Other(String),
 }
 
+impl Error {
+    pub fn code(&self) -> String {
+        // FIXME
```

Comment:
What's the plan with these TODOs?

Reply:
We need to make changes in typeql rust so that the error prefix is exposed — it's currently private, which is a bug —, along with the formatted error code, and then we can have these accessors implemented cleanly.